pyuvsim/simsetup.py

```diff
@@ -207,28 +229,14 @@ def initialize_uvdata_from_params(obs_params):
         which_ants = antnames[np.where(beam_ids == beamID)]
         for a in which_ants:
             beam_dict[a] = beamID
-        uvb = UVBeam()
-        if beam_model in ['gaussian', 'uniform']:
-            # Identify analytic beams
-            if beam_model == 'gaussian':
-                try:
-                    beam = pyuvsim.AnalyticBeam('gaussian', sigma=telparam['sigma'])
-                except KeyError as err:
-                    print("Missing sigma for gaussian beam.")
-                    raise err
-            else:
-                beam = pyuvsim.AnalyticBeam('uniform')
-            beam_list.append(beam)
-            continue
-        if not os.path.exists(beam_model):
-            filename = beam_model
-            path = os.path.join(SIM_DATA_PATH, filename)
-            if not os.path.exists(path):
-                raise OSError("Could not find file " + filename)
-        else:
-            path = beam_model   # beam_model = path to beamfits
-        uvb.read_beamfits(path)
-        beam_list.append(uvb)
+        if beam_model == 'gaussian':
+            try:
+                sigma = telparam['sigma']
+                beam_model = beam_model + '_' + str(sigma)
+            except KeyError:
+                print("Missing sigma for gaussian beam.")
+                raise err
```

Comment:
why not just raise a value error here? In general, I think we should avoid print statements for warnings and errors -- instead use the warnings package and raise the appropriate error types.